src/components/Picking.vue

```diff
@@ -1,96 +1,26 @@
 <template>
-    <body>
-        
-        <v-btn @click="toggleView()">Toggle View</v-btn>
+  <div>
+    <v-btn @click="toggleView()">Toggle View</v-btn>
+    <choose-menu v-if="view === 0"></choose-menu>
 
-
-    <div id ="randomArray">
-        <h1>Picking<v-select label="Select" 
-            :items="['Dinner', 'Date']"
-            variant="solo-filled"
-            ></v-select></h1>
-
-    <!-- <div>Selected: {{choice.cateory}}</div> -->
-<!-- 
-            <select v-model="choice.cateory">
-                <option disabled value="">Please select one</option>
-                <option>Breakfast</option>
-                <option>Lunch</option>
-                <option>Dinner</option>
-                <option>Date</option>
-            </select>
- -->
-
-                <v-btn @click="pickDinner()">Choose Dinner</v-btn>
-                <v-btn @click="pickDate()">Choose Date</v-btn>
-
-        <h3>Tonight's Dinner: {{ tonightDinner }}</h3>
-        <h3>Tonight's date: {{tonightOption}}</h3>
-
-    </div>
-
-    <div id ="myDiv">
-        <h1>This is toggling display</h1>
-
-        <v-text-field
-        @keydown.enter="addDinner(input)"
-        label="add your own favourite options!"
-        v-model="input"
-    ></v-text-field>
-                <v-btn @click="addDinner(input)">Add your favourite meals</v-btn>
-
-    </div>
-
-    </body>
-   
+    <add-options v-if="view === 1"> </add-options>
+  </div>
 </template>
 <script setup>
-import { ref,  computed } from 'vue'
-import { set } from '@vueuse/core'
+import { ref, computed } from "vue";
+import { set } from "@vueuse/core";
+import AddOptions from "./AddOptions.vue";
+import ChooseMenu from "./ChooseMenu.vue";
 
 // reactive state
-const input = ref('')
-const dinner = ['chicken', 'death', 'kebabs']
-const date = ['movie', 'restaurant', 'arcade']
-var tonightDinner = ref(' ')
-var tonightOption = ref(' ')
-
 
+const view = ref(0);
```

Comment:
since you only have two views you could consider using a boolean instead of 0 and 1 here
for example view = ref(true) 
then on your v-if you can just write :v-if="view"

Reply:
For scalability leaving as integer